avaframe/com1DFA/com1DFA.py

```diff
@@ -384,11 +397,12 @@ def com1DFACore(cfg, avaDir, cuSimName, inputSimFiles, outDir, simHash=""):
     particles, fields, dem, reportAreaInfo = initializeSimulation(
         cfg, outDir, demOri, inputSimLines, cuSimName
     )
+    nPartInitial = particles["nPart"]
 
     # ------------------------
     #  Start time step computation
-    Tsave, particlesList, fieldsList, infoDict = DFAIterate(
-        cfg, particles, fields, dem, inputSimLines, simHash=simHash
+    Tsave, infoDict, contDictXY = DFAIterate(
```

Comment:
Minor thing: what does the contDictXY stand for? Maybe a more descriptive name would be good? I.e. what's in that dict?

Reply:
it is the x, y coordinates of the contour line - so **cont**our Line **dict**ionary **x, y** coordinates thereof

Reply:
Then i suggest contourDictXY -> I was not sure whether the cont meant continouus....

Reply:
renamed to